straxen/plugins/pulse_processing.py

```diff
@@ -222,7 +222,6 @@ class PulseProcessingHe(PulseProcessing):
         records_he=False,
         pulse_counts_he=True)
     depends_on = 'raw_records_he'
-    parallel = 'process'
```

Comment:
Why did you remove this? I am asking because it was previously the default for PulseProcessing. Hence I used it for the nveto version of the plugin as well.

Reply:
It's obscuring the fact that we are already copying this option from the parent so this change is not of any relevance for the nveto.